1.sql_select.sql

```diff
@@ -0,0 +1,89 @@
+--case1 :  get all columns
+select * from table_name
+
+--case2 :  gel columns with alias
+select  name as Isim,
```

Comment:
should use English words instead of the Turkish words

Reply:
done!!